Customization.md

```diff
@@ -4,11 +4,14 @@ The default themeing is for HSL, but that might not be what you need. Here's ins
 
 ## API backend.
 
-The GraphQL API endpoint is defined in `src/index.tsx`.
+The GraphQL API endpoint is defined in `src/monitorConfig.js`.
+
+## monitorConfig.js 
+ MonitorConfig.js hold all the relevant information for configuring virtual monitor's appereance and incoming data. Agency string is meant for  `src/ui/Logo.tsx` to change correct logo depending on url where the user is going to. uri is the  GraphQL API endpoint.
 
 ## Logo
 
-Logo is in `src/ui/Logo.tsx` which by default just re-exports `src/ui/HslLogo.tsx`. You can create a new file for a new logo and have `src/ui/Logo.tsx` import that instead. The logo can be either JSX or an image tag or whatever you might want.
+Logo is in `src/ui/Logo.tsx` which by default returns the logo depending on agency. You can create a new file for a new logo and have `src/ui/Logo.tsx` import that too. The logo can be either JSX or an image tag or whatever you might want.
```

Comment:
Strictly speaking agency is not the correct term here. OpenTripPlanner returns the stops in feedId:gtfsId format. Therefore, feedId would be more appropriate term (just slightly more difficult more OTP specific terminology)

Reply:
This is fine. I was wondering if the agency term would have been more clear for this project. I added little example to documentation to make it more clear.